src/cli/commands/test_commands/deploy.js

```diff
@@ -83,6 +83,12 @@ exports.builder = yargs => {
       requiresArg: true,
       type: 'number',
     },
+    yaml: {
+      description: `${'Default:'.bold} ${'app.yaml'
+        .yellow}. Specify the base yaml file to use when deploying.`,
+      requiresArg: true,
+      type: 'number',
```

Comment:
Shouldn't this be `type: 'string'`?

Reply:
Yep - fixed. (I had that fixed in my local version...)